clients/include/testing_hbmv_batched.hpp

```diff
@@ -0,0 +1,373 @@
+/* ************************************************************************
+ * Copyright 2018-2020 Advanced Micro Devices, Inc.
+ *
+ * ************************************************************************ */
+
+#include "cblas_interface.hpp"
+#include "flops.hpp"
+#include "near.hpp"
+#include "norm.hpp"
+#include "rocblas.hpp"
+#include "rocblas_datatype2string.hpp"
+#include "rocblas_init.hpp"
+#include "rocblas_math.hpp"
+#include "rocblas_random.hpp"
+#include "rocblas_test.hpp"
+#include "rocblas_vector.hpp"
+#include "unit.hpp"
+#include "utility.hpp"
+
+template <typename T>
+void testing_hbmv_batched_bad_arg(const Arguments& arg)
+{
```

Comment:
Good point. If the future of the *_bad_arg tests is just for nullptrs for pointers and the handle, then I don't see the use of ever passing in arg. However, if we ever plan on using it for, say, unsupported enums or something like that then we might want to keep it as an argument since it would be easier to just simply edit the .yaml then tediously adding calls to the function, perhaps?

Reply:
But for this PR arg is definitely useless, as it is in many places across the testing suite in *_bad_arg functions.